### PR TITLE
Update to include Certificate Chain info

### PR DESCRIPTION
**Pull request by bergerhoffer**
Replacement for PR #19042.

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@pbmoses Can you please review this and make sure that the changes coming in are as expected? We'll use this PR then in place of your original PR (which I'm closing). Thanks!

**Comment by bergerhoffer** [comment]
@sanchezl Can you please review and make sure this is accurate?

**Comment by bergerhoffer** [comment]
@xingxingxia or @barleyer - Can one of you please review this update? Thanks!

**Comment by pbmoses** [comment]
20763 LGTM (looks good to me) 

thank you!

File: modules/customize-certificates-api-add-named.adoc [hunk 1]
```diff
@@ -18,6 +18,17 @@ client's URL.
...
 
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file. This can be achieved with the following example: `cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem`.
+
+The order of the chain should be:
+
+* {product-title} master host certificate.
+
+* Intermediate CA certificate.
+
+* Root CA certificate.
```
Comment (sanchezl) [inline modules/customize-certificates-api-add-named.adoc]:
```suggestion
If a certificate chain is required to certify the server certificate, then the certificate chain must be appended to the server certificate. For example: 

----
$ cat server_cert.pem int2ca_cert.pem int1ca_cert.pem rootca_cert.pem>combined_cert.pem.
----

When combining certificates, the order of the certificates is important. Each following certificate must directly certify the certificate preceding it, for example:

1. {product-title} master host server certificate.

2. Intermediate CA certificate that certifies the server certificate.

3. Root CA certificate that certifies the intermediate CA certificate.
```

**Review by sanchezl — COMMENTED** [review]
Tweaked the wording a little bit. There might be more than one intermediate certificate and I wanted to make it clear what exactly the ordering was.

**Comment by xingxingxia** [comment]
@bergerhoffer , talked with @wangke19 , he will help review when having time chance later.

**Review by pbmoses — APPROVED** [review]

File: modules/customize-certificates-api-add-named.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/customize-certificates-api-add-named.adoc]: Thanks @sanchezl - I've incorporated your suggested changes. Other than the numbering formatting, the only other thing I changed was to remove a period at the end of your `cat` command after `combined_cert.pem`. Please let me know if that period was required for some reason. Thanks!

Comment (wangke19) [reply modules/customize-certificates-api-add-named.adoc]: @bergerhoffer , A supplement to the certificate extension，added just like below,

If a certificate chain is required to certify the server certificate, then the certificate chain must be appended to the server certificate. Certificate files must be Base64 PEM-encoded and typically have a .crt or .pem extension. For example
...

Comment (bergerhoffer) [reply modules/customize-certificates-api-add-named.adoc]: Thanks @wangke19 - I'll add the additional sentence!

**Pull request merged**